pages/positions/addliquidity.tsx

```diff
@@ -244,6 +244,8 @@ const AddLiquidity: NextPage = () => {
         setIsOpen={setIsOpen}
         setTokenSelected={tokenSelected === 'tokenA' ? setTokenASelected : setTokenBSelected}
         otherToken={tokenSelected === 'tokenA' ? tokenBSelected : tokenASelected}
+        prevTokenA={tokenASelected}
```

Comment:
You just need otherTokenSelected={tokenSelected === 'tokenA' ? tokenBSelected : tokenASelected}

Do not need prevTokenA or prevTokenB

Reply:
Done